addons/addon-base-raas/packages/base-raas-workflow-steps/lib/steps/start-sagemaker-environment/start-sagemaker-environment.js

```diff
@@ -101,14 +101,6 @@ class StartSagemakerEnvironment extends StepBase {
 
   async getSageMakerService() {
     const [aws] = await this.mustFindServices(['aws']);
-    // increase retry defaults to reduce the risk of failures caused
```

Comment:
Why was this part removed?

Reply:
Ah! This was a mix up, I was trying to remove this part from start-sagemaker-enviornment-sc.js. Good catch!